tools/validators/instance_validator/validate/instance_parser.py

```diff
@@ -33,7 +33,19 @@
 """Schema separately parses translation to account for multiple valid formats
 github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md
 #defining-translations"""
-_TRANSLATION_SCHEMA = syaml.Str() | syaml.Any()
+_TRANSLATION_SCHEMA = syaml.Str() | syaml.MapPattern(
+    syaml.Str(),
+    syaml.Str() | syaml.Map({'present_value': syaml.Str(),
```

Comment:
same here.

Reply:
the first element str() is for the case when `translation: compliant`.
Strictyaml dont let you do `string_value` | Map thats why it is generic str() here then we do a validation at the end